src/Riok.Mapperly/Configuration/MapperConfiguration.cs

```diff
@@ -1,101 +1,85 @@
-using Microsoft.CodeAnalysis;
 using Riok.Mapperly.Abstractions;
-using Riok.Mapperly.Descriptors;
-using Riok.Mapperly.Helpers;
 
 namespace Riok.Mapperly.Configuration;
 
 public class MapperConfiguration
```

Comment:
Add a comment what exactly this class does and why it is almost equal to the MapperAttribute except for the nullability.

Reply:
Added a comment.